draft-ion.adoc

```diff
@@ -208,79 +200,176 @@ represented as follows:
+"OPTIONAL" in this document are to be interpreted as described {uri-rfc2119}[RFC2119] when they appear in UPPERCASE.  If these words are used without being spelled in uppercase then they are
+to be interpreted with their normal natural language meanings, without any normative meaning.
+
 
+[[content-structure]]
+== Content Structure
 
+Ion content MUST have exactly one root JSON object, called the _Root Object_.  The Root Object MAY contain
+name/value pairs called 'members' and any level of nested JSON objects.
+
+[[metadata]]
 == Metadata
 
-An Ion resource MAY contain a direct child _Meta Object_. If present,
-the _Meta Object_ represents data about its immediate parent Ion resource.
-The _Meta Object_ is OPTIONAL.
+An JSON object MAY contain a _Meta Object_ member named `meta`. If present, the _Meta Object_ represents data about its immediate parent JSON object. The _Meta Object_ is OPTIONAL.
+
+An Ion parser MUST identify a member as a _Meta Object_ if and only if:
 
-If a _Meta Object_ is present, the resource field name MUST equal the
-case-sensitive octet sequence `meta` and the field value MUST be a non-null, non-empty
-JSON object. A _Meta Object_ MUST contain one or more name/value
-pairs. Ion parsers MUST reject Ion resources where a direct child `meta` field
-exists and the field value is `null` or an empty JSON object.
+1. The member name is equal to the case-sensitive octet sequence `meta` and
+2. The member value is a JSON Object that has one or more members.
 
-The field names within a _Meta Object_ MUST be unique; Ion parsers MUST either
-reject _Meta Object_ fields with duplicate field names or use a JSON
-parser that returns only the lexically last duplicate field, as specified
+If a member name is equal to the case-sensitive octet sequence `meta` and the member value is NOT a JSON Object (i.e. it is an array, number, string or the octet sequence literals `null`, `false`, or `true` as defined by {uri-rfc-json-value}[RFC 7159, Section 3]), an Ion parser MUST NOT identify the member as a _Meta Object_.
+
+If a member name is equal to the case-sensitive octet sequence `meta` and the member value is an empty JSON Object with no members, an Ion parser MUST NOT identify the member as a _Meta Object_.
+
+If a member name is not equal to the case-sensitive octet sequence `meta`, an Ion parser MUST NOT identify a member as a _Meta Object_.
+
+The member names within an identified _Meta Object_ MUST be unique; Ion parsers MUST either
+reject _Meta Object_ members with duplicate member names or use a JSON
+parser that returns only the lexically last duplicate member, as specified
 in {uri-ecmaScript-5-1-15-12}[Section 15.12 (The JSON Object)] of {uri-ecmaScript-5-1}[ECMAScript 5.1].
 
-The set of fields that a _Meta Object_ must contain to be considered valid
+The set of members that a _Meta Object_ must contain to be considered valid
 is resource-dependent.  In the absense of any resource-specific requirements,
```

Comment:
I think I might say this like:

A _Meta Object_ MAY contain members that are not defined in Ion. Any member not understood by implementations SHOULD be ignored.

I would leave out of the spec any concept of the meaning of the resource or its domain, which is how the current wording feels to me.